SwrveSDK/src/amazon/java/com/swrve/sdk/SwrveEmpty.java

```diff
@@ -18,4 +19,12 @@ protected SwrveEmpty(Context context, String apiKey) {
     //@Override
     //public void setPushNotificationListener(ISwrvePushNotificationListener pushNotificationListener) {
     //}
+
+    @Override
+    public void iapPlay(String productId, double productPrice, String currency, String purchaseData, String dataSignature) {
```

Comment:
iapPlay is for Google play, not needed in this flavour. It is probably forcing you because it might be in an interface or similar, remove